sticky_notes.py

```diff
@@ -81,3 +81,45 @@ def list_stickynotes(self, event):
                 self.delay_task(index,
                                 self.sender(format_note(index, note), event))
             return first_note
+
+    @cmd
+    def del_stickynote(self, event):
+        args = event.args
+        from_nick = event.meta['nick']
+
+        if len(args) < 1:
+            return "Missing nick argument."
+
+        if len(args) < 2:
+            return "Missing stickynote number argument."
+
+        if len(args) >= 2:
```

Comment:
@JakubNvk is this if necessary? Given the lines above we can be sure that `len(args) >= 2` is indeed true.